nulib/meadow
# Terraform preservation bucket replication for production environment only

# Summary 
Adds Terraform resources for the buckets, policies and rules required to support Cross-Region Replication from `meadow-p-preservation` in `us-east-1` to `meadow-p-preservation-replica` in `us-west-1`.

# Specific Changes in this PR
- Terraform code to provide replication functionality

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
Ingest a file in Meadow, and check to see if the preservation object has been replicated to `meadow-p-preservation-replica`.

Also please let developers know if there are any special instructions to test this in the development environment. 

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
